workbench-core/infrastructure/README.md

```diff
@@ -37,7 +37,7 @@ Next, set the properties and initialize the construct.
 // If a user pool name or user pool client name are not provided, CDK will generate them for you.
 const cognitoProps: WorkbenchCognitoProps = {
   domainPrefix = 'myDomainPrefix',
-  websiteUrl = 'https://mysite.mydomain.com',
+  websiteUrls = ['https://mysite.mydomain.com'],
```

Comment:
not related to this line;; but as a customer why do I care about this change? Could you update the PR description and add it to the future changelog file

Reply:
added to PR description, will add to change files once review is complete